### ISSUE TITLE
The underlying Jackson parser doesn't support BigInteger

### ISSUE THREAD

**Issue by effiban** [jsurfer/JsonSurfer#73]
Hi,
In our project we need to deserialize Json objects containing very large integers (greater than max of `long`).
I would expect the parser to be able to handle such an integer and produce a `BigInteger`, however instead we get an exception:

```
com.fasterxml.jackson.core.JsonParseException: Numeric value (13744120747940549538) out of range of long
 at [Source: UNKNOWN; line: 1, column: 58197]
```

It appears that a custom deserializer is needed, but there doesn't seem to be any way to define it, or else I couldn't find the documentation.

(NOTE - we are actually using [Alpakka's JSON support](https://github.com/akka/alpakka/blob/master/json-streaming/src/main/scala/akka/stream/alpakka/json/impl/JsonStreamReader.scala) which uses JSurfer internally, so this might require a corresponding  change on their end as well).

Any thoughts?
Thanks

**Comment by effiban** [comment]
@yuliarotenshtein - FYI

**Comment by wanglingsong** [comment]
this pull request #38 might help but it's not merged yet. I will evaluate it again since many people request this feature

**Comment by wanglingsong** [comment]
@effiban the code is merged. It will be available in version 1.6.3

**Comment by effiban** [comment]
@wanglingsong - that's great, thanks!
